packages/vats/src/proposals/resetRoundIds.js

```diff
@@ -0,0 +1,47 @@
+import { E } from '@endo/far';
+import { makeStorageNodeChild } from '@agoric/internal/src/lib-chainStorage.js';
+import { makeMarshal } from '@endo/marshal';
+import { Fail } from '@agoric/assert';
+
+/**
+ * @param {BootstrapPowers & {
+ *   consume: { chainStorage: any };
+ * }} powers
+ */
+export const resetRoundId = async ({
+  consume: { chainStorage, chainTimerService },
+}) => {
+  const TOKENS = ['stTIA', 'stkATOM'];
+
+  const priceFeedStorageNode = await makeStorageNodeChild(
+    chainStorage,
+    'priceFeed',
+  );
+  for (const token of TOKENS) {
+    const name = `${token}-USD_price_feed`;
+    const tokenStorageNode = await makeStorageNodeChild(
+      priceFeedStorageNode,
+      name,
+    );
```

Comment:
I believe the node is in a subchild called `latestRound`

Reply:
thanks. Fixed.